Brain/src/router/api.py

```diff
@@ -492,4 +499,36 @@ def delete_data(data: AutoTaskDelete):
             200, "Deleted data from real-time database of firebase", ""
         )
 
+    """@generator.request_body(
+            {
+                "token": "String",
+                "uuid": "String",
+                "contactIds": [
+                    String
+                ]
+            }
+        )
+
+        @generator.response(
+            status_code=200, schema={"message": "message", "result": "test_result"}
+        )
+
+        """
+
+    @router.post("/contacts/getByIds")
+    def getContactsByIds(data: GetContactsByIds):
```

Comment:
Please follow the naming rule of method in python. Instead of it, it's better named get_cotnacts_by_ids()